### ISSUE TITLE
Terminal Cursor : Ability to customize it

### ISSUE THREAD

**Issue by isontheline** [isontheline/pro.webssh.net#602]
# Feature description
- [x] Style: block, pipe-symbol, underline
- [x] Blink or no blink
- [ ] ~~Multiple sizes (half block vs. full block, wide line vs. narrow line)~~

**Comment by joeledwards** [comment]
I would find this especially helpful for vim sessions. I noticed that the cursor maintains the state it had when I started typing. When blinking, if it is visible when I start typing, it remains visible. If it is not visible when I start typing, it remains invisible. When in vim, and trying to navigate, this means I have to wait for the cursor to appear before navigating.

Being able to disable the blink feature would fix this for me.

An alternative is to use a similar approach I have seen in other terminal emulators, which makes the cursor visible as soon as a key is pressed.
